cheetah.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Vikas(Vikas_2_0)",
+			"oem": "Google",
+			"device": "Pixel 7 Pro",
+			"filename": "crDroidAndroid-15.0-20241106-cheetah-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/cheetah/11.x/crDroidAndroid-15.0-20241106-cheetah-v11.0-BETA.zip/download",
+			"timestamp": 1730873149,
+			"md5": "8dcd2afec13c774594eef5a5562339c0",
+			"sha256": "de3e66455d6e47259349df158f40a3a0b34117dfb5ec7cba09951beb1165eb6d",
+			"size": 1584139211,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-official-cheetah-15-0-crdroidandroid-v11-x.4702073/",
```

Comment:
as has been discussed few times already, please update forum post as per latest template
footer is bad